### ISSUE TITLE
zrepl status:  "terminal entry not found"

### ISSUE THREAD

**Issue by henningkessler** [zrepl/zrepl#204]
I get "terminal entry not found" if I run `zrepl status` on `pull` or `source` on a running and working installation. Source is FreeBSD 11.3p2 and Pull is FreeNAS 11.2u5. What am I doing wrong ;-)

Regards

**Comment by problame** [comment]
It's likely from the `tcell` library: https://github.com/gdamore/tcell/blob/1922b3786eaefecbba6fd8e11dbfd3531fad7a54/terminfo/terminfo.go#L37

tcell 1.2 fixes https://github.com/gdamore/tcell/issues/252 which seems related to your problem

Could you please provide the output of `echo "$TERM"` on your systems? if it's `xterm-color`, I'm pretty sure we hit https://github.com/gdamore/tcell/issues/252

**Comment by henningkessler** [comment]
output of `echo "$TERM"` is indeed `xterm-color` following the instructions in your hint to [gdamore/tcell#252](https://github.com/gdamore/tcell/issues/252) I deleted 

```export TERM=xterm-color```  

in my `.bashrc`. This changed the output to 

```> echo $TERM xterm-256color```

and now everything looks good. Happiness :-)

**Comment by problame** [comment]
Please revert that change to your bashrc and try the build c4be60c (see the green checkmark above for binary builds from the CI).

**Comment by henningkessler** [comment]
I installed zrepl from pkg and I am afraid this a bit over my head or it will at least take me a while to compile this build from source but I would try a binary If you would willing to share it with me. Sorry for the hassle...

**Comment by problame** [comment]
![image](https://user-images.githubusercontent.com/956573/62688248-c454cb80-b9c8-11e9-9548-163dfcdf5a9e.png)

**Comment by henningkessler** [comment]
OH SHAME ON ME ;-). Now I got it. The build is working! Thank you for your patience.

**Comment by problame** [comment]
All right, so the build is working with `export TERM=xterm-color`?

**Comment by henningkessler** [comment]
Yes this build is working with `export TERM=xterm-color`.

**Comment by henningkessler** [comment]
Sorry for the mess. I forget to reload the Terminal. After logging out and back in I get the the following error: 
```exec: "infocmp": executable file not found in $PATH```

**Comment by problame** [comment]
See https://github.com/gdamore/tcell/issues/252#issuecomment-501921074
I guess you can `pkg install ncurses` which provides `infocmp`.
@woodsb02 this is an edge case, so I wouldn't add ncurses as a hard dependency of the package on FreeBSD, would you?

**Comment by henningkessler** [comment]
At least on the FreeNAS (the `PULL` side) installing ncurses this is not really an option. I guess getting rid `export TERM=xterm-color` is the better alternative, right?

**Comment by problame** [comment]
Yes, you don't have another option.
What is the default $TERM on FreeNAS?
If zrepl doesn't work out of the box on FreeNAS, this is an issue we should definitely addres

**Comment by henningkessler** [comment]
default $TERM in FreeNAS seams to be csh.

**Comment by problame** [comment]
That seems odd, given `csh` is a shell...

**Comment by henningkessler** [comment]
I guess I should definitely stop try to multitask and concentrate more. Default $TERM is `xterm-256color`.

**Comment by problame** [comment]
OK, so I guess default $TERM works out of the box.

**Comment by henningkessler** [comment]
Yes

**Comment by problame** [comment]
OK, the fix commited in c4be60c is unacceptable, reopening this.
We need infocmp inside tmux because tmux sets `$TERM` to `screen`.
A lot of people are going to use zrepl in tmux, and I don't want the dependency on ncurses for now.

For the record:

**FreeNAS 11.2-U6 installation, 11.2 Jail created, no packages installed**

tcell 1.2 works out of the box, no need to install ncurses

```
root@zrepl:~ # which infocmp
infocmp: Command not found.
root@zrepl2:~ # echo $TERM
xterm-256color
```

**FreeBSD 12.0-STABLE VM IMAGE no packages installed, root user**

tcell 1.2 works out of the box, no need to install ncurses

```
root@freebsd:~ # which infocmp
infocmp: Command not found.
root@freebsd:~ # echo $TERM
xterm-256color
```

**With tmux installed, on either system**

```
root@freebsd:~ # zrepl status
exec: "infocmp": executable file not found in $PATH
root@freebsd:~ # echo $TERM
screen
```
